app/javascript/app/pages/ndc-sdg/ndc-sdg-component.jsx

```diff
@@ -29,7 +31,13 @@ class NdcSdg extends PureComponent {
           href={location.href}
         />
         <NdcsSdgsMetaProvider />
-        <Header size="small" route={route} className={styles.header}>
+        <Header
+          size="small"
+          route={route}
+          className={cx(styles.header, {
+            [styles.isOpen]: qs.parse(location.search).goal !== ''
```

Comment:
I don't think that is responsibility of the component, the container is the one who takes care of theses thing passing props to the component keeping it "dumb"
So I'd move this logic to the container and pass it as `isOpen` or any other name prop to the component